nucliadb_node/src/bin/writer.rs

```diff
@@ -98,6 +95,9 @@ async fn main() -> NodeResult<()> {
     let (shutdown_notifier, shutdown_notified) = get_shutdown_notifier();
     let shard_cache = Arc::new(ShardWriterCache::new(settings.clone()));
 
+    lifecycle::initialize_writer(settings.clone())?;
+    lifecycle::initialize_merger(Arc::clone(&shard_cache), settings.clone())?;
```

Comment:
I think we also want to initialize the merger on the bindings initialization, don't we?